crypto/x509/x509_cmp.c

```diff
@@ -74,6 +74,11 @@ int X509_CRL_cmp(const X509_CRL *a, const X509_CRL *b)
     return X509_NAME_cmp(a->crl.issuer, b->crl.issuer);
 }
 
+int X509_CRL_is_equal(const X509_CRL *a, const X509_CRL *b)
+{
+    return X509_NAME_is_equal(a->crl.issuer, b->crl.issuer);
+}
+
```

Comment:
I strongly disagree with the name of this function.

Reply:
So what is your suggestion then?